features/api/backend_usages/edit.feature

```diff
@@ -13,11 +13,11 @@ Feature: Product > Integration > Backends > Edit
     And follow "My API"
     And press "Integration" within the main menu
     And follow "Backends" within the main menu
-    And follow "Edit config with Backend 1"
-    Then the current page is the edit usage config page between "My API" and "Backend 1"
+    And follow "Edit config with Backend 1" to the edit backend usage page of "Backend 1" for product "My API"
```

Comment:
I am really sorry @akostadinov I should have been more attentive on the meeting yesterday.
But can you please clarify again why this is needed? :sweat_smile: 

I think I understood that the problem was with the links that were wrong. So, the test clicked on the link that had the specified name, and indeed the browser went to that URL, but the page returned a 404 Not Found. However, the last step passed because indeed the path is correct.

Was this the issue? Or was it something else?

I think that probably what we need to do is just assert that after clicking on that link and reloading the page we can see "Edit Backend Usage" and "Backend 1" on the page - that would verify that we actually navigated to the correct page.

Then this new `"to {this path}"` part would be optional, I think.

But maybe I misunderstood the issue.

Anyway, I'm OK with merging this if @josemigallas is convinced :sweat_smile:

Reply:
@mayorova , explanation is here https://github.com/3scale/porta/pull/3677#discussion_r1486764000

If the target link is unexpected, then we fail on the page where that link is present so we see the source page in artifacts where we can understand more easily what was wrong. Instead of seeing a 404 or whatever wrong target page in artifacts and having no clue why we went to an unexpected URL.